libretro/LibretroGraphicsContext.cpp

```diff
@@ -143,3 +145,61 @@ LibretroGraphicsContext *LibretroGraphicsContext::CreateGraphicsContext() {
 	ctx->Init();
 	return ctx;
 }
+
+std::vector<u32> TranslateDebugBufferToCompare(const GPUDebugBuffer *buffer, u32 stride, u32 h) {
```

Comment:
Hm, this shouldn't be needed anymore right?

Reply:
The previous "software software" is still there as none-context, and it would not work at all otherwise.